app/controllers/my_controller.rb

```diff
@@ -81,23 +82,30 @@ def change_password
   # Administer access tokens
   def access_token; end
 
-  # Configure user's mail notifications
+  # Configure user's in app notifications
   def notifications
     render html: '',
            layout: 'angular',
            locals: { menu_name: :my_menu }
   end
 
+  # Configure user's mail reminders
+  def reminders
+    render html: '',
+           layout: 'angular',
+           locals: { menu_name: :my_menu }
+  end
+
   # Create a new feeds key
   def generate_rss_key
-    if request.post?
-      token = Token::RSS.create!(user: current_user)
-      flash[:info] = [
-        t('my.access_token.notice_reset_token', type: 'RSS').html_safe,
-        content_tag(:strong, token.plain_value),
-        t('my.access_token.token_value_warning')
-      ]
-    end
+    token = Token::RSS.create!(user: current_user)
+    flash[:info] = [
+      # rubocop:disable Rails/OutputSafety
+      t('my.access_token.notice_reset_token', type: 'RSS').html_safe,
```

Comment:
We can avoid calling `.html_safe` by chaniging the i18n key to something with a `*_html` suffix. Not meant as a review feedback for this PR, just as a reminder for us.